airflow/plugins/operators/external_table.py

```diff
@@ -41,9 +41,11 @@ def _bq_client_create_external_table(
             len(source_objects) == 1
         ), "cannot use hive partitioning with more than one URI"
         opt = bigquery.external_config.HivePartitioningOptions()
+        # _Strongly_ recommend using CUSTOM mode and explicitly-defined
+        # key schema for more than a trivial number of files
         opt.mode = hive_options.get("mode", "AUTO")
         opt.require_partition_filter = hive_options.get(
-            "require_partition_filter", False
+            "require_partition_filter", True
```

Comment:
Ok so just to confirm I could override this, for example for small Airtable data?

Reply:
yes! for safety's sake I wanted to make it the default, but it's very over-rideable in the yml